cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -1,26 +1,22 @@
-require 'open3'
+require 'kontena/plugin_manager'
 
 module Kontena::Cli::Plugins
   class UninstallCommand < Kontena::Command
     include Kontena::Util
     include Kontena::Cli::Common
+    include Kontena::PluginManager::Common
 
     parameter 'NAME', 'Plugin name'
 
-    option "--force", :flag, "Force remove", default: false, attribute_name: :forced
+    def uninstaller
+      Kontena::PluginManager::Uninstaller.new(name)
+    end
 
     def execute
-      exit_with_error "Plugin #{pastel.cyan(name)} is not installed" unless Kontena::PluginManager.instance.installed(name) && !ENV['NO_PLUGINS']
-      confirm unless forced?
-      spinner "Uninstalling plugin #{pastel.cyan(name)}" do |spin|
-        begin
-          Kontena::PluginManager.instance.uninstall_plugin(name)
-        rescue => ex
-          $stderr.puts pastel.red("#{ex.class.name} : #{ex.message}")
-          logger.error(ex)
-          spin.fail
-        end
+      exit_with_error "Plugin #{name} has not been installed" unless installed?(name)
```

Comment:
Yes, it used to just crash, but it was fixed by just warning if a plugin fails to load and continue loading, and it's in this PR too.